packages/material-ui/src/TableBody/TableBody.test.js

```diff
@@ -12,8 +12,11 @@ describe('<TableBody />', () => {
   const render = createClientRender();
 
   function mountInTable(node) {
-    const wrapper = mount(<table>{node}</table>);
-    return wrapper.find('table').childAt(0);
+    const utils = render(<table>{node}</table>);
```

Comment:
You can return these directly in all the table test and query the component under test with `container.querySelector(tagName)`.